### DISCUSSION TITLE
使用 vue3 toRefs 出現 binding value must be a string or number

### DISCUSSION THREAD

**Discussion by dinofree0618** [element-plus/element-plus#2533] [Q&A]
ElementPlusError: [Util] binding value must be a string or number

請問我是哪裡給值錯了?還是尚未規範所導致的?
v-model我很肯定都是對的

**Comment by michaeljaekel** [comment]
As an workaround (if you don't want to update to the newest beta version https://blog.csdn.net/hjg719/article/details/118803564) you can set label-position and label-width.
e.g.
<el-form label-position="top" label-width="100px"></el-form>
